src/lib/dialog/dialog.spec.ts

```diff
@@ -128,6 +128,40 @@ describe('MdDialog', () => {
     expect(overlayContainerElement.querySelector('md-dialog-container')).toBeFalsy();
   });
 
+  describe('disableClose option', () => {
+    it('should prevent closing via clicks on the backdrop', () => {
+      let config = new MdDialogConfig();
+      config.viewContainerRef = testViewContainerRef;
+      config.disableClose = true;
+
+      dialog.open(PizzaMsg, config);
+
+      viewContainerFixture.detectChanges();
+
+      let backdrop = <HTMLElement> overlayContainerElement.querySelector('.md-overlay-backdrop');
```

Comment:
FYI, we're going to start preferring using the `as HTMLElement` syntax over this style cast. 
(Which I still need to document in our style guide)